Fix/proofread

patrickvacek opened the pull request:
Started proofreading for version 1.1. I'm not done but this is what I've got for today. Mostly just minor stuff at this point with a few rewordings that might help clarify things.

jhdalek55 commented:
Thanks, Patrick. I was planning to set aside Wednesday or Thursday to do a complete pass myself. So, you doing a pass through is great. I feel like I need to set aside an entire day to do both documents and that easier said then done. I'll read through these later today.

jhdalek55 commented on uptane-standard.md:
```diff
@@ -692,13 +692,13 @@ The vehicle version manifest is a metadata structure that MUST contain the follo
-  * The vehicle's unique identifier (e.g. the VIN)
-  * The Primary ECU's unique identifier (e.g. the serial number)
+  * The vehicle's unique identifier (e.g., the VIN)
+  * The Primary ECU's unique identifier (e.,g. the serial number)
```
comma goes after the "g", not the "e."

patrickvacek replied:
Oops. Fixed!

jhdalek55 reviewed (changes requested):
fix one comma that looks like this e.,g.

jhdalek55 reviewed (approved):
Good catch.

jhdalek55 reviewed (approved):
These all appear good. I had flagged an earlier point about commas inside quotes. It is standard in U.S. English to have commas inside quotes. I believe though this was questioned earlier, though I can't remember the context . I am content at this point to be consistent with whatever style is dominant on this point....meaning if commas are mostly outside, lets bring the few inside in line.

patrickvacek commented on uptane-standard.md:
```diff
@@ -169,13 +169,13 @@ These instructions specify the components necessary for a compliant implementati
 The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in {{RFC2119}}.
 
-In order to be considered “Uptane-compliant,” an implementation MUST follow all of these rules as specified in the document.
+In order to be considered "Uptane-compliant", an implementation MUST follow all of these rules as specified in the document.
```
Ah yes. I forgot that's one common US-Americanism that I just cannot stand. How about I just drop the quotation marks? I don't think they are necessary anyway. Otherwise I'll revert the comma change.

jhdalek55 replied:
Perfect resolution. Drop the quotation marks.

iramcdonald commented:
Hi,

Yes please drop the quotes here.

Global Editorial:  The enclosing of somewhat arbitrary terms in double
quotes should be
avoided entirely in the Uptane Standard.  The meaning (emphasis?) is
ambiguous and
no apparent value is added by this double quotes use.

Cheers,
- Ira

*Ira McDonald (Musician / Software Architect)*

*Chair - SAE Trust Anchors and Authentication TF*
*Co-Chair - TCG Trusted Mobility Solutions WG*

*Co-Chair - TCG Metadata Access Protocol SG*








*Chair - Linux Foundation Open Printing WGSecretary - IEEE-ISTO Printer
Working GroupCo-Chair - IEEE-ISTO PWG Internet Printing Protocol WGIETF
Designated Expert - IPP & Printer MIBBlue Roof Music / High North
Inchttp://sites.google.com/site/blueroofmusic
<http://sites.google.com/site/blueroofmusic>http://sites.google.com/site/highnorthinc
<http://sites.google.com/site/highnorthinc>mailto: blueroofmusic@gmail.com
<blueroofmusic@gmail.com>(permanent) PO Box 221  Grand Marais, MI 49839
906-494-2434*


On Wed, Nov 18, 2020 at 3:55 AM Patrick Vacek <notifications@github.com>
wrote:

> *@patrickvacek* commented on this pull request.
> ------------------------------
>
> In uptane-standard.md
> <https://github.com/uptane/uptane-standard/pull/184#discussion_r525913925>
> :
>
> > @@ -169,13 +169,13 @@ These instructions specify the components necessary for a compliant implementati
>
>
>
>  The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in {{RFC2119}}.
>
>
>
> -In order to be considered “Uptane-compliant,” an implementation MUST follow all of these rules as specified in the document.
>
> +In order to be considered "Uptane-compliant", an implementation MUST follow all of these rules as specified in the document.
>
>
> Ah yes. I forgot that's one common US-Americanism that I just cannot
> stand. How about I just drop the quotation marks? I don't think they are
> necessary anyway. Otherwise I'll revert the comma change.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/uptane/uptane-standard/pull/184#discussion_r525913925>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE33UO7YG2HSNOL6DGAEIX3SQODYPANCNFSM4TXOZZUA>
> .
>

patrickvacek commented:
Okay, I've finished my proofread scan and have addressed the concerns raised so far. I'm going to open a few issues for things I can't resolve myself, though.

jhdalek55 reviewed (approved):
I think "Standards" shows up in these early pages as a reference what the document provided (i.e. standard forms of action for implementing the program) as opposed to the document. But, clearly it makes sense that we should be referring to the document here. Good catch and let's definitely fix it.

jhdalek55 commented:
@patrickvacek I think I have reviewed and approved ll your proposed commits and flagged just a few minor things. Ira has reviewed some of these commits. Do you think we could merge this new before it becomes too cumbersome?

patrickvacek commented:
> I think I have reviewed and approved all your proposed commits and flagged just a few minor things. Ira has reviewed some of these commits. Do you think we could merge this new before it becomes too cumbersome?

Yep, let's merge it! I don't have the rights, so if someone who does doesn't mind pressing the right buttons...

iramcdonald commented:
Hi,

Thanks Patrick for a bunch of good editorial cleanups and Lois for keeping
this moving!

*Ira McDonald (Musician / Software Architect)*

*Chair - SAE Trust Anchors and Authentication TF*
*Co-Chair - TCG Trusted Mobility Solutions WG*

*Co-Chair - TCG Metadata Access Protocol SG*








*Chair - Linux Foundation Open Printing WGSecretary - IEEE-ISTO Printer
Working GroupCo-Chair - IEEE-ISTO PWG Internet Printing Protocol WGIETF
Designated Expert - IPP & Printer MIBBlue Roof Music / High North
Inchttp://sites.google.com/site/blueroofmusic
<http://sites.google.com/site/blueroofmusic>http://sites.google.com/site/highnorthinc
<http://sites.google.com/site/highnorthinc>mailto: blueroofmusic@gmail.com
<blueroofmusic@gmail.com>(permanent) PO Box 221  Grand Marais, MI 49839
906-494-2434*


On Thu, Nov 19, 2020 at 3:19 AM Patrick Vacek <notifications@github.com>
wrote:

> I think I have reviewed and approved all your proposed commits and flagged
> just a few minor things. Ira has reviewed some of these commits. Do you
> think we could merge this new before it becomes too cumbersome?
>
> Yep, let's merge it! I don't have the rights, so if someone who does
> doesn't mind pressing the right buttons...
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/uptane/uptane-standard/pull/184#issuecomment-730208543>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE33UO7SPWPP7YEIAXN6BTTSQTIKDANCNFSM4TXOZZUA>
> .
>
